openupgrade_scripts/scripts/stock/14.0.1.1/upgrade_analysis.txt

```diff
@@ -89,6 +88,7 @@ DEL ir.model.access: stock.access_stock_picking_type_portal
 DEL ir.model.access: stock.access_stock_rule
 DEL ir.model.access: stock.access_stock_rule_manager
 DEL ir.model.access: stock.access_stock_warehouse_orderpoint_portal
+DEL ir.model.constraint: stock.constraint_stock_production_lot_name_ref_uniq
```

Comment:
We must remove this constraint manually in the `stock` migration scripts.

Reply:
PR of `stock` is not merged. I will update that PR when this one is merged.